host/frontend/host-orchestrator/orchestrator/instancemanager_test.go

```diff
@@ -200,31 +200,26 @@ func TestProcedureExecuteInnerStageWithFails(t *testing.T) {
 
 func TestLaunchCVDProcedureBuilder(t *testing.T) {
 	cvdBuildID := "1"
-	cvdBin := "bin/cvd"
+	paths := IMPaths{
```

Comment:
What do you think of making these absolute paths?

Reply:
done. definitely better for consistency purposes.